src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonUtilities.Schema.Create.cs

```diff
@@ -617,11 +632,23 @@ private static bool TypeIsIntegerWithStringNumberHandling(AIJsonSchemaCreateCont
                     switch (type)
                     {
                         case "integer" or "number":
+                            if (numericType is not null)
```

Comment:
Is this change related, or is it just flagging an existing conflict that we were previously missing? Is it still a conflict if numericType == type?

Reply:
It's something we were missing and got hit with the newly added tests. I don't think we need to account for duplicate numeric types since we don't generate that.